<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>425</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>51</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>425</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Circle Launcher 2.0 Beta</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralWidget {
	background-color: rgb(255, 255, 255);
}</string>
   </property>
   <widget class="QWidget" name="topWidget" native="true">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>402</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(45, 45, 45);
font: 10pt &quot;Arial&quot;;
color: #AAAAAA;
QPushButton {
	background-color: transparent;
	border: none;
}

QPushButton:hover {
	background-color: rgb(76, 76, 76);
}</string>
    </property>
    <widget class="QPushButton" name="mainButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>51</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#mainButton {
	color: white;
	font-weight: bold;
	background-image: url(:/img/orange-header.png);
	border: none;
}

QPushButton:hover{
	background-color: rgb(76, 76, 76);
}
</string>
     </property>
     <property name="text">
      <string>main</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addonButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>0</y>
       <width>51</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: transparent;
	border: none;
}

QPushButton:hover{
	background-color: rgb(76, 76, 76);
}</string>
     </property>
     <property name="text">
      <string>addons</string>
     </property>
    </widget>
    <widget class="QPushButton" name="toolButton">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>0</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {	
	background-image: url(:/img/toolButton.png);
	background-color: transparent;
	border: none;
}

QPushButton:hover{
	background-color: rgb(76, 76, 76);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="playButton">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>0</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {	
	background-image: url(:/img/play16x16.png);

	background-color: transparent;
	border: none;
}

QPushButton:hover{
	background-color: rgb(76, 76, 76);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>338</x>
       <y>0</y>
       <width>3</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QComboBox" name="realmCombo">
     <property name="geometry">
      <rect>
       <x>254</x>
       <y>0</y>
       <width>81</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox
{
	background-color: transparent;
	border:none;
	padding:1px;
	text-align: right;
}

QComboBox::drop-down
{
	border-width:1px;
}

QComboBox::down-arrow
{
	image: url(:/img/drop-down.png);
}
QComboBox:hover {
	background-color: transparent;
	background-color: rgb(76, 76, 76);
}
</string>
     </property>
    </widget>
    <widget class="QPushButton" name="arenaButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>0</y>
       <width>41</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	
	background-color: transparent;
	border: none;
}

QPushButton:hover{
	background-color: rgb(76, 76, 76);
}</string>
     </property>
     <property name="text">
      <string>arena</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="mainWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>27</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#mainWidget{
background-color: #1a0f08;
	background-color: rgb(255, 255, 255);
}</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>mainButton</tabstop>
  <tabstop>addonButton</tabstop>
  <tabstop>arenaButton</tabstop>
  <tabstop>realmCombo</tabstop>
  <tabstop>toolButton</tabstop>
  <tabstop>playButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
